src/cli/peakdetector/PeakDetectorCLI.cpp

```diff
@@ -556,11 +556,13 @@ void PeakDetectorCLI::loadCompoundsFile() {
 		int loadCount = DB.loadCompoundCSVFile(mavenParameters->ligandDbFilename);
 		mavenParameters->compounds = DB.compoundsDB;
 		if (loadCount == 0) {
-			cerr << "Warning: Given compound database is empty!" << endl;
-		} else {
+			cerr << "Warning: Uploaded compound database is empty!" << endl;
+		}
+		else{
 			cout << "Total Compounds Loaded : " << loadCount << endl;
 		}
-	} else {
+	}
+	 else {
```

Comment:
We have recently had discussions around some code style guidelines and we agreed that we were going to stick to the "`else` after closing `if` braces" and not on the next line. The style guide is not final yet so its fine until then (unless this also a rebase issue and was unintentional).